src/components/SecondaryNav/SecondaryNav.css

```diff
@@ -0,0 +1,54 @@
+.SecondaryNav {
+    @apply
+      border-b-2
+      relative
+      bg-white
+      overflow-hidden
+      border-mediumGrey;
+}
+
+.SecondaryNav ul {
```

Comment:
Noticed that I had set a max-width on `ul` in typography so you'll override it here so it doesn't wrap. That's my bad 🤦‍♂️ 
<img width="834" alt="Screen Shot 2019-04-16 at 4 25 19 PM" src="https://user-images.githubusercontent.com/1334131/56241675-7821fc00-6064-11e9-85ba-37fcdea41954.png">